ui/app/components/regex-validator.hbs

```diff
@@ -32,8 +32,6 @@
       <div>
         <Toggle
           @name={{concat @attr.name "-validation-toggle"}}
-          @status="success"
```

Comment:
there were no alternatives to the component, so `<Toggle>` now renders these classes by default